rust/Dockerfile

```diff
@@ -1,52 +1,76 @@
 # Global args to use in build commands
 ARG ALPINE_VERSION="3.18"
-ARG PACKAGE
 
-FROM lukemathwalker/cargo-chef:latest-rust-alpine${ALPINE_VERSION} as chef
```

Comment:
I decided to use our images because it's just +1 line of code but a smaller attack surface. We also won't be in a situation where we want an Alpine version (eg. with fixed CVE) for which `lukemathwalker/cargo-chef` don't have an image for.